packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.controller.js

```diff
@@ -0,0 +1,25 @@
+import { MANAGE_ARCHIVE_DOC_LINK } from '../../cold-archives.constants';
+
+export default class ColdArchiveContainersManageArchiveController {
+  /* @ngInject */
+  constructor($translate, coreConfig) {
+    this.$translate = $translate;
+    this.coreConfig = coreConfig;
+  }
+
+  $onInit() {
+    this.trackPage('containers::container::manage');
```

Comment:
Is it possible to use the `atInternet` property inside the routing so we could get rid of this ?
```javascript
atInternet: {
  rename: 'containers::container::manage',
},
```

Reply:
Done.